scripts/deploy.bash

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Deploy assets to DEPLOY_HOST.
+
+# Required environment variables:
+#   DEPLOY_KEY
+#   DEPLOY_USER
+#   DEPLOY_HOST
+#   DEPLOY_PATH
+#   DEPLOY_PORT
+
+# Halt on errors:
+set -e
+# Echo commands:
+set -x
+
+# SSH identity_file (DO NOT inadvertently rsync to production!)
+ID_FILE="ssh_deploy_key"
+
+KH_FILE="known_hosts"
+
```

Comment:
I may be paranoid, but be sure to delete these files again. Otherwise we risk that they get lost..

Reply:
clean way of creating a unique temp directory is
MYDIR=$(mktemp -d)
but be sure to clean it up, also in the event of failure.

Reply:
I'll delete them (the old workflow just left `~/id_rsa` lying around).